synthesizer/src/vm/finalize.rs

```diff
@@ -791,17 +792,66 @@ impl<N: Network, C: ConsensusStorage<N>> VM<N, C> {
                     //     }
                     // }
 
-                    // Initialize the stakers.
-                    let mut stakers = IndexMap::with_capacity(committee.members().len());
-                    // Iterate over the committee members.
-                    for (validator, (microcredits, _)) in committee.members() {
-                        // Insert the validator into the stakers.
-                        stakers.insert(*validator, (*validator, *microcredits));
+                    // Ensure that all members of the committee are open.
+                    ensure!(
+                        committee.members().iter().all(|(_, (_, is_open))| *is_open),
+                        "Ratify::Genesis(..) all members of the committee must be open"
+                    );
```

Comment:
Remove this. I'm not understanding why this design consideration was introduced.

Reply:
This check was introduced to be consistent with `credits.aleo`.
When a validator initially bonds through `bond_public`, they are default set to be open. See [here](https://github.com/AleoHQ/snarkVM/blob/2c3b775f9c869e8ba03d10e91b94d8bf8297d388/synthesizer/program/src/resources/credits.aleo#L129).